prisma/migrations/20230214191120_add_uname_to_community/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[uname]` on the table `Community` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `data` to the `Community` table without a default value. This is not possible if the table is not empty.
```

Comment:
We might need to drop the prod DB and rerun this if there are already communities in the DB?